NU5129 warning for System.Resources.Extensions package

When building the runtime repo with the latest build of the .NET 8 Preview 7 SDK, the following warning  occurs:

```
/vmr/.dotnet/sdk/8.0.100-preview.7.23369.1/Sdks/NuGet.Build.Tasks.Pack/buildCrossTargeting/NuGet.Build.Tasks.Pack.targets(221,5): warning NU5129: - At least one .targets file was found in 'buildTransitive/', but 'buildTransitive/System.Resources.Extensions.targets' was not. [/vmr/src/runtime/artifacts/source-build/self/src/src/libraries/System.Resources.Extensions/src/System.Resources.Extensions.csproj]
/vmr/.dotnet/sdk/8.0.100-preview.7.23369.1/Sdks/NuGet.Build.Tasks.Pack/buildCrossTargeting/NuGet.Build.Tasks.Pack.targets(221,5): warning NU5129:  [/vmr/src/runtime/artifacts/source-build/self/src/src/libraries/System.Resources.Extensions/src/System.Resources.Extensions.csproj]
```

--- a/src/libraries/System.Resources.Extensions/src/System.Resources.Extensions.csproj
+++ b/src/libraries/System.Resources.Extensions/src/System.Resources.Extensions.csproj
@@ -50,7 +50,8 @@ System.Resources.Extensions.PreserializedResourceWriter</PackageDescription>
 
   <Target Name="GeneratePackageTargetsFile" 
           Inputs="$(MSBuildAllProjects)"
-          Outputs="$(SuggestedBindingRedirectsPackageFile)">
+          Outputs="$(SuggestedBindingRedirectsPackageFile)"
+          Condition="'$(NetFrameworkMinimum)' != ''">
     <PropertyGroup>
       <SuggestedBindingRedirectsPackageFileContent><![CDATA[<Project>
   <!-- ResolveAssemblyReferences will never see the assembly reference embedded in the resources type,

